functions/dashboard.js

```diff
@@ -13,183 +12,212 @@ function getMaxRainChance(forecast) {
     forecast.chanceOfRain.T18_24,
   ];
 
-    const maxPercentage = Math.max(
-  ...rainChances.map(p => {
-    const percentage = parseInt(p);
-    return Number.isNaN(percentage) ? -Infinity : percentage;
-  })
-);    
-
-    return maxPercentage;
-
-} 
+  const maxPercentage = Math.max(
+    ...rainChances.map((p) => {
+      const percentage = parseInt(p);
+      return Number.isNaN(percentage) ? -Infinity : percentage;
+    })
+  );
 
+  return maxPercentage;
+}
 
 async function getWeather() {
-    const response = await fetch('https://weather.tsukumijima.net/api/forecast/city/015010');
-    const data = await response.json();
-    return data;
+  const response = await fetch(
+    "https://weather.tsukumijima.net/api/forecast/city/015010"
```

Comment:
釧路高専の所在地が釧路なので、釧路の天気予報を出した方がいいかな

```suggestion
    "https://weather.tsukumijima.net/api/forecast/city/014020"
```

Reply:
なぜ室蘭…